src/visualization/src/MeshcatVisualizer.cpp

```diff
@@ -265,6 +260,18 @@ bool MeshcatVisualizer::setModelState(const iDynTree::MatrixView<const double> &
                                modelName);
 }
 
+bool MeshcatVisualizer::setSphere(const std::string &name,
+                                  const iDynTree::Position &position,
+                                  const double radius,
+                                  const iDynTree::VectorFixSize<3> &color)
+{
+    MeshcatCpp::Material m;
+    m.set_color(color[0] * 255, color[1] * 255, color[2] * 255);
+    MeshcatCpp::Meshcat meshcat;
+    meshcat.set_object(name, MeshcatCpp::Sphere(radius), m);
+    // meshcat.set_transform(name, iDynTree::Transform(iDynTree::Rotation::Identity(), position));
```

Comment:
It is intended that this is commented out?

Reply:
Outdated, sorry, my bad.